### ISSUE TITLE
F# compiler error message contains odd characters

### ISSUE THREAD

**Issue by tmat** [dotnet/cli#7345]
## Steps to reproduce

Create an F# library with following code

```F#
namespace Lib

module Say =    
    let hello =
        printf "%s" 1
```

Run 
```
dotnet build
```

## Expected  behavior

```
C:\Temp\f#\Lib>dotnet build
Microsoft (R) Build Engine version 15.3.409.57025 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  CSLib -> C:\Temp\f#\CSLib\bin\Debug\netstandard2.0\CSLib.dll
C:\Temp\f#\Lib\Library.fs(5,21): error FS0001: This expression was expected to have type 'string' but here has type 'int' [C:\Temp\f#\Lib\Lib.fsproj]

Build FAILED.
```

## Actual behavior

![image](https://user-images.githubusercontent.com/41759/28852321-1c1ae286-76dd-11e7-9846-48e8d9530e87.png)

## Environment data
```
.NET Command Line Tools (2.0.0-preview3-006910)

Product Information:
 Version:            2.0.0-preview3-006910
 Commit SHA-1 hash:  ad12b539a7

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.14393
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\2.0.0-preview3-006910\

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0
  Build    : e8b8861ac7faf042c87a5c2f9f2d04c98b69f28d
```

**Comment by cartermp** [comment]
cc @KevinRansom 

Yeah, that definitely looks odd.  I can repro as well.

**Comment by livarcocc** [comment]
Fixed in 2.0.0.
